cms/models.py

```diff
@@ -845,6 +858,60 @@ def serve(self, request, *args, **kwargs):
         raise Http404
 
 
+class CertificateIndexPage(RoutablePageMixin, Page):
+    """
+    Certificate index page placeholder that handles routes for serving
+    certificates given by UUID
+    """
+
+    parent_page_types = ["HomePage"]
+    subpage_types = []
+
+    slug = CERTIFICATE_INDEX_SLUG
+
+    @classmethod
+    def can_create_at(cls, parent):
+        """
+        You can only create one of these pages under the home page.
+        The parent is limited via the `parent_page_type` list.
+        """
+        return (
+            super().can_create_at(parent)
+            and not parent.get_children().type(cls).exists()
+        )
+
+    @route(r"^([A-Fa-f0-9-]+)/?$")
+    def course_certificate(
```

Comment:
I guess, it would be `bootcamp_certificate`

Reply:
Updated.